network/p2p/p2p.go

```diff
@@ -71,8 +73,11 @@ type p2pNetwork struct {
 	backoffConnector *libp2pdiscbackoff.BackoffConnector
 	subnets          []byte
 	libConnManager   connmgrcore.ConnManager
-	nodeStorage      operatorstorage.Storage
-	operatorPKCache  sync.Map
+
+	nodeStorage        operatorstorage.Storage
+	operatorPKCache    sync.Map
```

Comment:
don't we have this cache in validation already? we should probably use the same cache all over.

Reply:
and maybe the best will be to move the cache into the storage itself struct to provide all node with it.
unless there's a good reason to keep these separated

Reply:
Seems like two different caches, renamed to make it clearer.